src/actions/posts.js

```diff
@@ -875,10 +878,19 @@ export async function getProfilesAndStatusesForPosts(posts, dispatch, getState)
     // Do this after firing the other requests as it's more expensive
     const usernamesToLoad = getNeededAtMentionedUsernames(state, posts);
 
+    let emojisToLoad;
+    if (getConfig(state).EnableCustomEmoji === 'true') {
+        emojisToLoad = getNeededCustomEmojis(state, posts);
+    }
+
     if (usernamesToLoad.size > 0) {
         promises.push(getProfilesByUsernames(Array.from(usernamesToLoad))(dispatch, getState));
     }
 
+    if (emojisToLoad && emojisToLoad.size > 0) {
```

Comment:
We shouldn't need the `emojisToLoad` check in the if statement, as it is declared to exist 6 lines up.

Reply:
It might not get set though, such as when custom emojis are disabled, so we only want to push the promise when it is set

Reply:
Of course. My bad, it is declared, not defined. Just testing you. ;)